src/neuroglancer/datasource/deepzoom/frontend.ts

```diff
@@ -0,0 +1,304 @@
+    this.url = url.substring(0, url.lastIndexOf('.')) + '_files';
+  }
+
+  getSources(volumeSourceOptions: VolumeSourceOptions) {
+    const modelResolution = this.info.scales[0].resolution;
+    const {rank} = this;
+    return transposeNestedArrays(this.info.scales.map(scaleInfo => {
+      const {resolution} = scaleInfo;
+      const stride = rank + 1;
+      const chunkToMultiscaleTransform = new Float32Array(stride * stride);
+      chunkToMultiscaleTransform[chunkToMultiscaleTransform.length - 1] = 1;
+      const {lowerBounds: baseLowerBound, upperBounds: baseUpperBound} =
+          this.info.modelSpace.boundingBoxes[0].box;
+      const lowerClipBound = new Float32Array(rank);
+      const upperClipBound = new Float32Array(rank);
+      for (let i = 0; i < 3; ++i) {
+        const relativeScale = resolution[i] / modelResolution[i];
+        chunkToMultiscaleTransform[stride * i + i] = relativeScale;
+        const voxelOffsetValue = scaleInfo.voxelOffset[i];
+        chunkToMultiscaleTransform[stride * rank + i] = voxelOffsetValue * relativeScale;
+        lowerClipBound[i] = baseLowerBound[i] / relativeScale - voxelOffsetValue;
+        upperClipBound[i] = baseUpperBound[i] / relativeScale - voxelOffsetValue;
+      }
+      if (rank === 4) {
+        chunkToMultiscaleTransform[stride * 3 + 3] = 1;
+        lowerClipBound[3] = baseLowerBound[3];
+        upperClipBound[3] = baseUpperBound[3];
+      }
+      return makeDefaultVolumeChunkSpecifications({
+               rank,
+               dataType: this.dataType,
+               chunkToMultiscaleTransform,
+               upperVoxelBound: scaleInfo.size,
+               volumeType: this.volumeType,
+               chunkDataSizes: scaleInfo.chunkSizes,
+               baseVoxelOffset: scaleInfo.voxelOffset,
+               volumeSourceOptions,
+             })
+          .map((spec): SliceViewSingleResolutionSource<VolumeChunkSource> => ({
+                 chunkSource: this.chunkManager.getChunkSource(DeepzoomImageTileSource, {
+                   credentialsProvider: this.credentialsProvider,
+                   spec,
+                   parameters: {
+                     url: resolvePath(this.url, scaleInfo.key),
+                     encoding: scaleInfo.encoding,
+                     overlap: this.info.overlap,
+                     tilesize: this.info.tilesize
+                   }
+                 }),
+                 chunkToMultiscaleTransform,
+                 lowerClipBound,
+                 upperClipBound,
+               }));
+    }));
+  }
+}
+
+interface DZIMetaData {
+  width: number;
+  height: number;
+  tilesize: number;
+  overlap: number;
+  format: string;
+}
+
+function getDZIMetadata(
+    chunkManager: ChunkManager, credentialsProvider: SpecialProtocolCredentialsProvider,
+    url: string): Promise<DZIMetaData> {
+  if (url.endsWith('.json') || url.includes('.json?'))
+    throw new Error('DZI-JSON: OpenSeadragon hack not supported yet.');
```

Comment:
add curly braces, also please add link to format specification for the dzi-json format

Reply:
That's a hack by OpenSeadragon guys, http://openseadragon.github.io/examples/tilesource-dzi/ - they just rewrote the XML as JSON, even keeping the `xmlns` ("XML namespace") part. While it's not standard, it's trivial to support, just I don't have ready-made test data for it (theirs is not CORS enabled).
I think for now I'll add the curly braces only.

Reply:
Done (-ish). So it's not implemented, but link is provided with a comment, and curly braces are there.